### DISCUSSION TITLE
Explorer Patcher only partially functional with Windows 11 updates

### DISCUSSION THREAD

**Discussion by DrMartyG** [valinet/ExplorerPatcher#1559] [General]
With new Windows 11 updates, my Explorer patcher is only partially functional, and tries repeatedly to open without success. My start menu is still Windows 10, but the Task Bar is Windows 11.  I'm not very techy, and am frustrated with some other issues I've had that may be related to the Windows 11 updates and Explorer Patcher incompatibility, and lost the ability to sign into my main user account, and had to sign in to the guest account which seemed to be totally Windows 11 with no Explorer Patcher at all, and do a system restore to get things back to partial Explorer Patcher functionality, and access my main user account.

**Comment by pyrates999** [comment]
Do the following:

1. uninstall explorer patcher
2. export this registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher" and upload it to here
3. delete the same registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
4. reboot
5. try logging into the account that you could not sign into before while explorer patcher was installed
6. install explorer patcher
7. verify you get a notification that symbols have been downloaded
8. restart windows explorer using either the task manager or in the properties window of explorer patcher

**Comment by vippium** [comment]
EP needs an update.. Wait for valinet to update it to the latest version. Currently, he is busy with #1108 . So, we need patience.
